src/core/mod.rs

```diff
@@ -0,0 +1,39 @@
+mod game;
+mod menu;
+
+pub use bevy::prelude::*;
+
+use crate::ui;
+
+pub(super) fn plugin(app: &mut App) {
+    // Setup state
```

Comment:
Personal pet peeve: Should be `Set up state` for the verb form, `Setup state` for the noun form :)

Reply:
I'll use State setup :p